tests/testthat/test-write.R

```diff
@@ -1,57 +1,76 @@
-test_that("SAS Transport file", {
+data_to_save <- dplyr::tibble(X = c(1, 2, NA), Y = c("a", "", "c"), Z = c(1, 2, 3))
```

Comment:
Can we use the minimal_test data that @averissimo created?  Also, I don't think we should make data outside of the test_that calls as it can spill into later tests.

Reply:
also i think it is okay not to be DRY here in our tests

Reply:
Yeah, I will work on it as a part of #96